kiwi/tasks/system_build.py

```diff
@@ -223,15 +216,14 @@ def process(self):                                      # noqa: C901
         setup.setup_plymouth_splash()
         setup.setup_timezone()
 
-        system.pinch_system(
-            manager=manager, force=True
-        )
         # make sure manager instance is cleaned up now
         del manager
 
         # setup permanent image repositories after cleanup
         setup.import_repositories_marked_as_imageinclude()
         setup.call_config_script()
+        system.pinch_system(manager=None, force=False)
+        system.pinch_system(manager=None, force=True)
```

Comment:
I think manager=None must not be set as this is the default value. A short comment would imho also make sense. Thus I think it could be simplified into

```python
# handle uninstall package requests, gracefully uninstall with dependency cleanup
system.pinch_system(force=False)

# handle delete package requests, force uninstall
system.pinch_system(force=True)
```